docs/installation.md

```diff
@@ -105,7 +105,7 @@ Or simply set up the environment variables as below in your source file, e.g. `~
 ```bash
 if [ -z ${PYTHONPATH+x} ]; then export PYTHONPATH=""; fi
 export MINTPY_HOME=~/tools/MintPy
-export PATH=${PATH}:${MINTPY_HOME}/mintpy/cli
+export PATH=${PATH}:${MINTPY_HOME}/src/mintpy/cli
 export PYTHONPATH=${PYTHONPATH}:${MINTPY_HOME}
```

Comment:
I think it might make more sense to set:
```
export MINTPY_HOME=~/tools/MintPy/src
```
as I believe that that's what you'll want to add to the `PYTHONPATH`.

Reply:
Yep, thanks!

Reply:
I just realize there is quite some usage of `$MINTPY_HOME/tests` and `$MINTPY_HOME/docs` in the repo, thus, keeping `MINTPY_HOME=~/tools/MintPy` seems more convenient and consistent. So I rolled back and kept the same MINTPY_HOME as before, and update the exports of PATH and PYTHONPATH here.